change/@fluentui-react-6c6ea224-d7e4-405c-92cc-8268da44ee7a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
You should either remove these change files for `@fluentui/react` or rename them to `office-ui-fabric-react`.